rules/opentitan.bzl

```diff
@@ -45,6 +46,8 @@ PER_DEVICE_DEPS = {
     "sim_dv": ["@//sw/device/lib/arch:sim_dv"],
     "fpga_cw305": ["@//sw/device/lib/arch:fpga_cw305"],
     "fpga_cw310": ["@//sw/device/lib/arch:fpga_cw310"],
+    # FIXME do we want to use different parameters or just aim for CW310 compatiblity?
```

Comment:
For now, QEMU ot-earlgrey only emulates Earlgrey on CW310. It is likely that another QEMU machine is required if another HW platform needs to be emulated.

Reply:
The FIXME here is just to highlight that we have the possibility to have a QEMU-specific [arch lib](https://github.com/lowRISC/opentitan/tree/master/sw/device/lib/arch). One benefit of having a specific is to change the `kDeviceType` so that libraries are aware that the code is compiled for QEMU and can change its behaviour. This is used for more efficient logging on dv targets for example.

If we wanted to have another HW platform, we could proceed like fpgas and have `sim_qemu_cw310`, `sim_qemu_cw340` and so on but this is not done in this commit.

Reply:
Ok, got it.